lib/rake/build.rake

```diff
@@ -211,3 +219,17 @@ end
 def calculate_apps_commit_hash
   RakeUtils.git_folder_hash(*apps_build_trigger_paths)
 end
+
+def get_npm_build_command
+  if CDO.optimize_webpack_assets
+    # Skip clean step in CI to take advantage of cached build artifacts
+    if ENV['CI']
+      'npm run build:dist'
+    else
+      # Run npm commands separately to ensure NODE_OPTIONS are set correctly for the build step.
+      'npm run clean; npm run build:dist'
```

Comment:
Why are we adding an `npm run clean` to non-CI environments as part of this change?

Reply:
Or is the idea that we're moving this here from `build-in-parallel`? If so, I'd definitely like to understand better how we're confident that those things are equivalent across all variations of our build processes